### DISCUSSION TITLE
nextcloud-aio-database and initial configuration

### DISCUSSION THREAD

**Discussion by ho16** [nextcloud/all-in-one#1524] [Questions] [answered]
Hello,
during the first setup of nextcloud - all in one. I am coming ot the intial setup  of nextcloud. There I hasve to enter the name of the database, the user and the passoword. 

Where do I get the information from?

Do I have to create the user, and database by hand in the database container? When yes, how?

Thanks for any help

**Comment by szaimen** [comment]
Hi, this should be configured automatically. 

As this looks like a hickup, please try to clean it properly and try again. See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance

**Reply by ho16** [reply]
Hello,

I already made several tries. But it does not work. 
It is always for the postgresql user name, password, database name and database host in the intial configuration of nextcloud.
I did not find a way to skip it. 

Then I tried t use the parameters from the database container. 
It dies not work either.

**Reply by szaimen** [reply]
So you've reset your instance peoperly and also cleaned up the volumes?

Can you then please share your Nextcloud container log from directly after a fresh install?

**Reply by ho16** [reply]
Yes, I have made a few attempts and after a problem cleaned all data according to the posting. 

I will send the log.
What do you need the master container log?

**Reply by szaimen** [reply]
No, the Nextcloud container log after a fresh install.

**Reply by ho16** [reply]
I know there error...  
The partition on the hard disk is full.... :( 

Is there a way to direkt the volumes, and images to another location?
I have a large hard disc partition with soft raid in a special area.

**Reply by szaimen** [reply]
There are these options: 
https://github.com/nextcloud/all-in-one#how-to-change-the-default-location-of-nextclouds-datadir

https://github.com/nextcloud/all-in-one#how-to-store-the-filesinstallation-on-a-separate-drive

**Comment by szaimen** [comment, marked as answer]
I just released a fix with v4.0.0 Beta which should make debugging of issues like this in the future a bit easier. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Comment by ho16** [comment]
Sorry for all of these stupid questions.
After moving all belox /var/lib/docker to another device, it was working. 
I just had to adjust the jason file with the default location of the docker data...
Now all is working as expected. 
Many thanks for the assistance,

**Reply by szaimen** [reply]
All fine! Good that everything is finally resolved and working!
